### PR TITLE
feat: initial commit

### PR DESCRIPTION
**Pull request by bennypowers**
## What I did
- rename and reorganize all the icons:
  1. delete the white and red variants
  2. keep the black variants
  3. the `a` variants are default
  4. the `b` variants are renamed *-alt.svg
  5. keep the 2 color variants (renamed *-duotone)
- remove eps, png, jpeg, etc. these should be generated by a build script and gitignored
generally, icons live in `${setName}/${iconName}.svg`, e.g. to put a 50% harvey ball on a page,
```html
<dl>
  <dt><rh-icon set="harvey" icon="50"></rh-icon></dt>
  <dd>Corresponds to <code>src/harvey/50.svg</code></dd>
</dl>
```

The exception to this is the standard icons which have one level of subdirectory. These will require some build magic, but ultimately it should work something like this:

```html
<strong>"standard" is the default icon set, so doesn't need to be specified</strong>
<dl>
  <dt><rh-icon icon="school"></rh-icon></dt>
  <dd>Corresponds to <code>src/standard/buildings/school.svg</code></dd>
  <dt><rh-icon icon="beaker"></rh-icon></dt>
  <dd>Corresponds to <code>src/standard/science/beaker.svg</code></dd>
</dl>
```
This will mean contributors must be careful to avoid name conflicts.

## TODO:
- [ ] build scripts that generate optimized svg, jpg, png, webp, and eps artifacts, including red and white variants
- [ ] ci/cd which adds artifacts to github releases
- [ ] documentation, readme, contributing.md, etc
- [ ] automation to detect name conflicts (git hooks?)
- [ ] linting/verifying monotone icons work properly (no `fill` or `fill="currentColor"`)
- [x] perhaps some by-hand optimization of svg sources?
- [x] add license file

### CODE REVIEW

**Comment by markcaron** [comment]
@bennypowers I'm ok with this structure. We might need to update the icon names throughout the other systems to match these renamed duplicates.

@libbylevi are you ok with the icon names and directory structure?

**Comment by libbylevi** [comment]
This looks good. We can make sure to avoid re-using icon names as we create new icons (and I think the school/beaker example you found was an error, that icon SHOULD be called "beaker").

You can exclude the two-color / duotone versions, they're not commonly used.

**Comment by bennypowers** [comment]
yes, @libbylevi confirming that was a typo in the example. I fixed that now.

I'm going to continue working on the checklist above then. Thanks for the feedback

**Comment by bennypowers** [comment]
@libbylevi can you check if the files in this PR were saved with the right pixel boundaries? @markcaron said there were some possible issues with decimal precision.

**Comment by bennypowers** [comment]
bumping previous question re decimal @libbylevi. Am I clear to proceed?

**Review by libbylevi — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@rhds/icons",
+  "version": "1.0.0",
+  "description": "Red Hat Icons",
+  "main": "icons.js",
+  "module": "icons.js",
+  "types": "icons.d.ts",
+  "type": "module",
+  "scripts": {
+    "build": "echo \"no build yet\"",
```
Comment (bennypowers) [inline package.json:10]: **PLEASE HOLD**

todo:

- [ ] optimize source svgs
- [ ] add SVGO config to automate above

**Comment by bennypowers** [comment]
Gonna merge and we'll open issues for the other stuff

**Pull request merged**